### PR TITLE
fix: brc-20 send form bug

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/5287960634).

See title

### CODE REVIEW

File: src/app/features/bitcoin-choose-fee/bitcoin-choose-fee-loader.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export function BitcoinChooseFeeLoader({ children, isLoading }: BitcoinChooseFeeLoaderProps) {
+  if (isLoading) {
+    return (
+      <Flex py="108px" justifyContent="center" alignItems="center" width="100%">
+        <LoadingSpinner />
+      </Flex>
+    );
+  }
+  return children;
+}
```
Comment (kyranjamie) [inline src/app/features/bitcoin-choose-fee/bitcoin-choose-fee-loader.tsx]: This component doesn't actually do any loading, so as not to confuse with other "loader pattern", suggest rename. Separate loading component is a nice idea though, alternatively could live in layout component?

File: src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx [hunk 2]
```diff
@@ -45,6 +45,8 @@ export function BrcChooseFee() {
     amount: Number(amount),
     recipient,
   });
+  const recommendedFeeRate = feesList[1]?.feeRate.toString() || '';
```
Comment (kyranjamie) [inline src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx:48]: maybe a fee rate enum would work here for better readability

**Review by kyranjamie — DISMISSED** [review]

File: src/app/features/bitcoin-choose-fee/bitcoin-choose-fee-loader.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/features/bitcoin-choose-fee/bitcoin-choose-fee-loader.tsx]: you mean put LoadingSpinner in Layout component?

**Comment by markmhendrickson** [comment]
Let's release now as a hotfix?

File: src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx (hunk 2 above)
Comment (alter-eggo) [reply src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx:48]: you mean change feesList to object to get values by fee type? 
`feesList[BtcFeeType.Standart]` ?

Comment (kyranjamie) [reply src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx:48]: yeah

Comment (alter-eggo) [reply src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx:48]: seems, this will lead to quiet a huge refactor for the hotfix

**Review by kyranjamie — APPROVED** [review]

**Comment by kyranjamie** [comment]
> Let's release now as a hotfix?

We can release through normal process, as there's nothing in dev that shouldn't go out

**Pull request merged**